docs/build.md

```diff
@@ -0,0 +1,7 @@
+# Building the Library
+
+Build this library with
+```bash
```

Comment:
Can you add something like `cd react-svg-pan-zoom/` ? Because if you come on this page from an example page, it is not clear that you need to perform this command on the root folder instead of the example folder.

Reply:
Okay, I made the change you requested